File: fizzbuzz.js
function fizzbuzz(n) {
  // console.log("This is fizzbuzz!")
  let output = [];

  for (let i = 1; i <= n; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
      output.push("fizzbuzz");
    } else if (i % 5 === 0) {
      output.push("buzz");
    } else if (i % 3 === 0) {
      output.push("fizz");
    } else {
      output.push(i);
    }
  }

  return output;
}

console.log(fizzbuzz(20));

module.exports = fizzbuzz;

File: fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func fizzbuzz(n int) []string {
	output := make([]string, 0, n)
	for i := 1; i <= n; i++ {
		switch {
		case i%15 == 0:
			output = append(output, "fizzbuzz")
		case i%5 == 0:
			output = append(output, "buzz")
		case i%3 == 0:
			output = append(output, "fizz")
		default:
			output = append(output, strconv.Itoa(i))
		}
	}
	return output
}

func main() {
	fmt.Println(fizzbuzz(20))
}
